packages/starknet-snap/src/utils/starknetUtils.ts

```diff
@@ -186,6 +186,21 @@ export const declareContract = async (
   });
 };
 
+export const getAccountNonce = async (
```

Comment:
```suggestion
export const getLatestAccountNonce = async (
```

Reply:
maybe we call it `getAccountLatestNonce` because the method we use says  

`Gets the nonce of the account with respect to a specific block` 

in our case we get `latest`. So we get the nonce of the account with respect to the latest block.

Reply:
@stanleyyconsensys